README.md

```diff
@@ -27,35 +27,49 @@ PRs should be posted _before_ class begins on Monday, March 4, 2024.
 
 ### React Challenge
 
-1. React Debugging: The Card Draw React application has three (3) errors that need to be fixed. Use the console and error messages to guide you to the solution.
+1. React Debugging: The Card Draw React application has three (3) errors that need to be fixed. Use the console and error messages to guide you to the solution. ✅
 
-2. React Feature: Add functionality to the `Return Cards and Shuffle Deck` button using the `shuffle` function to reset the Card Draw application to its original state.
+-1st error was adding fragements syntax <></> in the return on App.js
+-2nd error was adding the export default Card to card.js component
+-3rd error was using passing props hand to Card.js. i used the destructured {hand}
 
-3. React Concepts: Add comments to the seven (7) notations in the `drawCard` function describing in detail the functionality line of code BELOW the comment.
+2. React Feature: Add functionality to the `Return Cards and Shuffle Deck` button using the `shuffle` function to reset the Card Draw application to its original state.✅
 
-4. STRETCH (optional): Add stylings of your choice to the application. Ensure the styling does not interfere with the card draw functionality.
+-added logic to the function shuffle. only added setHand([]). added the onclick to the button on line 53 which when clicked takes you back to the original state which is an empty array. 
+
+3. React Concepts: Add comments to the seven (7) notations in the `drawCard` function describing in detail the functionality line of code BELOW the comment.✅
+
+4. STRETCH (optional): Add stylings of your choice to the application. Ensure the styling does not interfere with the card draw functionality. ✅
 
 ### Interview Practice Questions
 
 1. What are three (3) syntax differences between HTML and JSX?
 
--
--
--
+- class vs className to specify css classes
+- in JSX you use {} for embeded javascripts expressions/variables. allows dynamic rendering.
+- fragment syntax <></> allows grouping in JSX
 
 2. What is yarn? What file(s) are modified when you run the command yarn in your terminal?
 
+-yarn is a Javascript package manager. running it in terminal adds the dependencies and creates the node_modules. this will then allow you to run you app using yarn start. 
+
```

Comment:
Great answer, Ryan!  Just know that it allows your app to function, not just run yarn start. :)